Add Femgineer logo

dalrayes opened the pull request:
Adds bronze sponsor section
Adds Femgineer logo and description
Moves sponsorship blurb to bronze section
@jarmstrng

dalrayes committed:
Add Bronze sponsor section
Add Femgineer logo and description

jarmstrng commented:
can you add a `.gitignore` file and add `.DS_Store` to it, so that those two files aren't committed?

jarmstrng reviewed (changes requested):
it looks like you missed the changes to the trello card :upside_down_face: the logo needs to be in a section titled Childcare, not Bronze, and they don't get text

jarmstrng commented on index.html:
```diff
@@ -369,6 +369,15 @@ <h3 class="col-xs-12">Silver Level Sponsors</h3>
+					<p><a href="http://femgineer.com" target="_blank">Femgineer</a> is an education company that empowers forward-thinking techies of all backgrounds and promotes inclusivity in the tech industry.</p>
+				</div>
+			</div>
 			<div class="col-md-4">
```
Can we get the sponsor-us blurb with a `col-md-8` instead of 4, so that it fills the rest of the container?

dalrayes committed:
Add Bronze sponsor section
Add Femgineer logo and description

.DS_Store in gitignore

Update name to childcare sponsors

dalrayes committed:
Update to childcare sponsor

dalrayes commented:
@jarmstrng Added gitignore, updated to childcare sponsor and widened the 'sponsor us' box.

jarmstrng commented:
Hm, it looks like the .DS_Store files are still in the commit, can you remove them?

jarmstrng commented on index.html:
```diff
@@ -369,7 +369,16 @@ <h3 class="col-xs-12">Silver Level Sponsors</h3>
+				<div class="logo">
+					<a href="http://femgineer.com/" target="_blank"><img src="assets/img/logos/femgineer-logo.png" /></a>
+					<p><a href="http://femgineer.com" target="_blank">Femgineer</a> is an education company that empowers forward-thinking techies of all backgrounds and promotes inclusivity in the tech industry.</p>
+				</div>
```
Please remove this text, their sponsor level does not include it

dalrayes committed:
.DS_Store removed

dalrayes committed:
Removes sponsor description - not included

dalrayes commented:
@jarmstrng Deletes DS Store and sponsor description
